.github/workflows/test_all.yml

```diff
@@ -51,9 +51,9 @@ jobs:
         uses: codecov/codecov-action@v3
         with:
           flags: ${{ matrix.operating-system }},${{ matrix.python-version }},base,complete
-  test_components_with_extras:
+  test_all_extras:
```

Comment:
Note: this step does not use `all_extras`, rather this is the step that uses `forecasting`, `classification`, etc. optional dependencies.

Reply:
yes, I see. Apologies, I got confused by the naming, since "with extras" is very close to "all extras", semanticlly.